tools/build_config/Jenkinsfile

```diff
@@ -239,14 +241,30 @@ Boolean run_build_stage = (! env?.PR_LABELS =~ "do-not-build")
 Boolean run_test_stage = (! env?.PR_LABELS =~ "(do-not-build|do-not-test)")
 
 pipeline {
+
   agent {
     label env.AGENT
   }
 
+  environment {
+    MATLAB_VERSION = get_param('MATLAB_VERSION', get_matlab_release(env.JOB_BASE_NAME))
```

Comment:
a comment here or in the `get_param` function definition would make the behaviour clear: if the Jenkins parameter is passed that value is used, otherwise the default is applied